### PR TITLE
chore: add Trusting Social users list

### PR DESCRIPTION
**Pull request by cuong-ts**
Signed-off-by: Cuong Nguyen Duc <90603605+cuong-ts@users.noreply.github.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [x] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [x] Does this PR require documentation updates?
* [x] I've updated documentation as required by this PR.
* [x] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [x] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: USERS.md [hunk 1]
```diff
@@ -253,3 +253,4 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [Yubo](https://www.yubo.live/)
 1. [Zimpler](https://www.zimpler.com/)
 1. [ZOZO](https://corp.zozo.com/)
+1. [Trusting Social](https://trustingsocial.com/)
```
Comment (crenshaw-dev) [inline USERS.md:256]: @cuong-ts can you move this up for alphabetical order?

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]

**Review by cuong-ts — COMMENTED** [review]
fixing oder

File: USERS.md (hunk 1 above)
Comment (cuong-ts) [reply USERS.md]: fixed @crenshaw-dev

**Review by crenshaw-dev — APPROVED** [review]
Thanks @cuong-ts!

**Pull request merged**